extensions/control-plane/api/management-api/transfer-process-api/src/main/java/org/eclipse/edc/connector/api/management/transferprocess/TransferProcessApiController.java

```diff
@@ -70,16 +72,23 @@ public TransferProcessApiController(Monitor monitor, TransferProcessService serv
     @Path("request")
     @Override
     public List<JsonObject> queryTransferProcesses(JsonObject querySpecDto) {
-        var querySpec = jsonLd.expand(querySpecDto)
-                .compose(expanded -> transformerRegistry.transform(expanded, QuerySpecDto.class))
-                .compose(dto -> transformerRegistry.transform(dto, QuerySpec.class))
+
+        Function<Result<JsonObject>, Result<QuerySpec>> expandedMapper =
```

Comment:
why not just use `querSpecDto = ofNullable(querySpecDto).orElseGet(() -> QuerySpec.none())`? that would be a bit easier to read IMO

Reply:
Do you mean instead of using the full empty builder? `QuerySpec.Builder.newInstance().build()`?